<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="30" hostname="DESKTOP-DJ4FDME" failures="6" tests="6" name="Test(failed)" time="0.14" errors="0" timestamp="2024-03-20T05:34:11 IST">
  <testcase classname="testNGDemo.TestScenario8" name="forth_testcase" time="0.034">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at testNGDemo.TestScenario8.forth_testcase(TestScenario8.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- forth_testcase -->
  <testcase classname="testNGDemo.TestScenario3" name="second_testcase" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at testNGDemo.TestScenario3.second_testcase(TestScenario3.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- second_testcase -->
  <testcase classname="testNGDemo.TestScenario9" name="sixth_testcase" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at testNGDemo.TestScenario9.sixth_testcase(TestScenario9.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="testNGDemo.TestScenario" name="first_testcase" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at testNGDemo.TestScenario.first_testcase(TestScenario.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="testNGDemo.TestScenario4" name="third_testcase" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at testNGDemo.TestScenario4.third_testcase(TestScenario4.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- third_testcase -->
  <testcase classname="testNGDemo.TestScenario6" name="forth_testcase" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at testNGDemo.TestScenario6.forth_testcase(TestScenario6.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- forth_testcase -->
  <testcase classname="testNGDemo.TestScenario9" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="testNGDemo.TestScenario3" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="testNGDemo.TestScenario9" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="testNGDemo.TestScenario" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="testNGDemo.TestScenario3" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="testNGDemo.TestScenario8" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="testNGDemo.TestScenario8" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="testNGDemo.TestScenario6" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="testNGDemo.TestScenario4" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="testNGDemo.TestScenario4" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="testNGDemo.TestScenario4" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="testNGDemo.TestScenario9" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="testNGDemo.TestScenario3" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="testNGDemo.TestScenario9" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="testNGDemo.TestScenario" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="testNGDemo.TestScenario" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="testNGDemo.TestScenario6" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="testNGDemo.TestScenario6" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="testNGDemo.TestScenario3" name="fifth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="testNGDemo.TestScenario8" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="testNGDemo.TestScenario6" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="testNGDemo.TestScenario3" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="testNGDemo.TestScenario" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="testNGDemo.TestScenario4" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="testNGDemo.TestScenario8" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="testNGDemo.TestScenario8" name="first_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- first_testcase -->
  <testcase classname="testNGDemo.TestScenario" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="testNGDemo.TestScenario9" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="testNGDemo.TestScenario4" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="testNGDemo.TestScenario6" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
</testsuite> <!-- Test(failed) -->
